### ISSUE TITLE
Walk through

### ISSUE THREAD

**Issue by waitwhatwhen** [cipheras/cipherginx#4]
Hey mate,
Have you thought about recording a little walk through on how you get this up and running?

**Comment by cipheras** [comment]
Yeah, I can do it. I thought that readme will be sufficient as it is not that complex.

**Comment by waitwhatwhen** [comment]
Cheers mate! I’d really appreciate it.

**Comment by americanitunes101** [comment]
that would actually be awesome - it would really help me too. will you keep us updated with this idea, @cipheras? you legend.
